Stratis.Bitcoin/Features/BlockStore/LoopSteps/BlockStoreInnerStepContext.cs

```diff
@@ -33,30 +39,62 @@ public BlockStoreInnerStepContext Initialize(ChainedBlock nextChainedBlock)
             return this;
         }
 
+        /// <summary>
+        /// A queue of blocks to be downloaded
+        /// </summary>
         public Queue<ChainedBlock> DownloadStack { get; private set; }
         public BlockStoreLoop BlockStoreLoop { get; private set; }
+
+        /// <summary>
+        /// This is the ChainedBlock the inner step starts on
```

Comment:
You could remove "This is the" without losing anything here. Also feel free to use `<summary>short comment</summary>` as one-liner for fields and properties.